template/workflows/manifests/draft.yaml

```diff
@@ -70,3 +70,17 @@ variables:
       value: "default"
     description: "the Kubernetes namespace"
     versions: ">=0.0.1"
+  - name: "FLEET"
+    type: "boolean"
+    kind: "boolean"
+    default:
+      value: false
+    description: "specifies whether or not this is a fleet deployment"
+    versions: ">=0.0.1"
+  - name: "FLEET_NAME"
```

Comment:
For consistency I'd call this FLEETNAME (no underscore) here to match CLUSTERNAME